### PR TITLE
Add missing modules to the documentation

### PR DESCRIPTION
**Pull request by vasuraman11**

### CODE REVIEW

File: docs/config/inventory.lst [hunk 1]
```diff
@@ -3,12 +3,18 @@
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/custom_config.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/dns.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/external.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/fq_driver.xml"/>
```
Comment (esproul) [inline docs/config/inventory.lst]: This is a stratcond module and is already included in that section, so it is not needed here.

File: docs/config/inventory.lst [hunk 2]
```diff
@@ -3,12 +3,18 @@
...
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/ganglia.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/graphite_plain.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/graphite_tls.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/handoff_ingestor.xml"/>
```
Comment (esproul) [inline docs/config/inventory.lst]: This is a stratcond module and is already included in that section, so it is not needed here.

File: docs/config/inventory.lst [hunk 3]
```diff
@@ -31,17 +37,23 @@
...
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.resmon.sparkpost.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.resmon.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.smtp.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.ssh2.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.tcp.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.varnish.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/ping_icmp.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/postgres.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/prometheus.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/rabbitmq_driver.xml"/>
```
Comment (esproul) [inline docs/config/inventory.lst]: This is a stratcond module and is already included in that section, so it is not needed here.

File: docs/config/inventory.lst [hunk 4]
```diff
@@ -31,17 +37,23 @@
...
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/resolver_cache.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/reverse_check.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/selfcheck.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/snmp.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/snmptrap.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/ssh2.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/statsd.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/stomp_driver.xml"/>
```
Comment (esproul) [inline docs/config/inventory.lst]: This is a stratcond module and is already included in that section, so it is not needed here.

File: docs/config/inventory.lst [hunk 5]
```diff
@@ -31,17 +37,23 @@
...
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/test_ingestor.xml"/>
```
Comment (esproul) [inline docs/config/inventory.lst]: This is a stratcond module, but does not appear in that section. It should be added to the list of includes for stratcon modules.

File: docs/config/inventory.lst (hunk 1 above)
Comment (vasuraman11) [reply docs/config/inventory.lst]: done

File: docs/config/inventory.lst (hunk 2 above)
Comment (vasuraman11) [reply docs/config/inventory.lst]: done

File: docs/config/inventory.lst (hunk 3 above)
Comment (vasuraman11) [reply docs/config/inventory.lst]: done

File: docs/config/inventory.lst (hunk 4 above)
Comment (vasuraman11) [reply docs/config/inventory.lst]: done

File: docs/config/inventory.lst (hunk 5 above)
Comment (vasuraman11) [reply docs/config/inventory.lst]: done

**Pull request merged**